enzyme/test/lit.site.cfg.py.in

```diff
@@ -64,6 +64,7 @@ if len("@ENZYME_BINARY_DIR@") == 0:
 config.substitutions.append(('%eopt', emopt))
 config.substitutions.append(('%llvmver', config.llvm_ver))
 config.substitutions.append(('%FileCheck', config.llvm_tools_dir + "/FileCheck"))
+config.substitutions.append(('%llc', config.llvm_tools_dir + "/llc"))
```

Comment:
This shouldn't be needed, see other integration tests in the integration test dir linked above